pkg/volume/csi/csi_mounter.go

```diff
@@ -323,6 +324,7 @@ func (c *csiMountMgr) podAttributes() (map[string]string, error) {
 		"csi.storage.k8s.io/pod.namespace":       c.pod.Namespace,
 		"csi.storage.k8s.io/pod.uid":             string(c.pod.UID),
 		"csi.storage.k8s.io/serviceAccount.name": c.pod.Spec.ServiceAccountName,
+		"csi.storage.k8s.io/ephemeral":           strconv.FormatBool(c.csiVolumeMode == ephemeralVolumeMode),
```

Comment:
@kfox1111 should the presence of this new field in the PodInfo depend on the feature gate?

Reply:
Hmm... That is a good question...

It shouldn't hurt anything to have it always defined as it is, but following the principal that nothing should change if the gate is disabled, it probably should be behind the gate, yeah.

Reply:
I've made it optional, i.e. it only gets added when the feature gate is set.